org.flathub.flatpak-external-data-checker.yml

```diff
@@ -61,12 +61,12 @@ modules:
               - -DCMAKE_INSTALL_SYSCONFDIR:PATH=/etc
             sources:
               - type: archive
-                url: http://deb.debian.org/debian/pool/main/a/apt/apt_2.2.4.tar.xz
-                sha256: 6eecd04a4979bd2040b22a14571c15d342c4e1802b2023acb5aa19649b1f64ea
+                url: http://deb.debian.org/debian/pool/main/a/apt/apt_2.5.3.tar.xz
+                sha256: e5b14a37125b7caf9534aad0670e0686b516e2fe9d4d441408397bb498c5c10a
                 x-checker-data:
                   type: debian-repo
                   root: http://deb.debian.org/debian
-                  dist: bullseye
+                  dist: bookworm # the bullseye version (2.2.4) doesn't compile
```

Comment:
I think we should switch all the modules that pull sources from Debian to use `bookworm`

Reply:
Debian 12 Bookworm is however still in the "testing" channel.
I use it personally, plus "unstable", and find them rather stable, so it may be fine.
Will do the changes then.